Pod/Classes/Categories/PINImage+ScaledImage.m

```diff
@@ -0,0 +1,53 @@
+//
+//  UIImage+ScaledImage.m
+//  Pods
+//
+//  Created by Michael Schneider on 2/9/17.
+//
+//
+
+#import "PINImage+ScaledImage.h"
+
+static inline PINImage *PINScaledImageForKey(NSString * __nullable key, PINImage * __nullable image) {
+    if (image == nil) {
+        return nil;
+    }
+    
+#if PIN_TARGET_IOS
```

Comment:
We should probably assert if image.CGImage is nil here, it isn't guaranteed to be non-nil (though it always should be if created by PINRemoteImage)